STEERING-COMMITTEE.md

```diff
@@ -263,12 +264,13 @@ In order to transition from the original, organization-appointed Steering
 Committee structure (the "bootstrap" committee), elections will be phased in
 over 14 months.
 
-In October 2020, four positions will be elected, including three Contributing
-seats and one End-User seat. One Google, one IBM/RedHat, and one VMware seat
-will be retained from the current Steering Committee.
+In October 2020, two positions will be elected, including two Contributing
```

Comment:
Is the size of the committee actually five then? I think we should make this consistent with the characterization of the committee's size above.

Reply:
Yes, the committee would be 5 until we figure out the two end-user seats. If that turns out to be too small, the committee could always amend the charter to add more seats.

Reply:
See line 132 for the change to 5 seats.